README.md

```diff
@@ -35,6 +35,20 @@ In order to comply with XACML restrictions placed on objects, a hook is used to
 
 ![image](https://cloud.githubusercontent.com/assets/2371345/9816201/d7e9a1e6-5871-11e5-90a0-51381eaf8fcb.png)
 
+### Drush 
+
+Apply XACML policy to target object. For example:
```

Comment:
Make this a header rather than the first sentence in a paragraph. Basically structure I'm thinking is like this...

**Apply a XACML policy to one target object**

To add policy.xml to object islandora:57: `drush -v --user=1 islandora_xacml_editor_apply_policy --policy=/tmp/policy.xml --pid=islandora:57`

To apply this policy to islandora:57 and all child objects, add the `--traversal` option.

**Force XACML inheritance to child objects**

To apply the XACML policy from islandora:root to its children:

`drush -v --user=1 islandora_xacml_editor_force_policy_inheritance --pid=islandora:root`

To apply this policy only to immediate children, use the `--shallow_traversal` option.

Reply:
Regarding the `--traversal` and `--shallow_traversal` options:
- For `islandora_xacml_editor_apply_policy` I assume if you don't include one of these, it applies to the target object only, correct? And is `--shallow_traversal` an option for this command?
- For `islandora_xacml_editor_force_policy_inheritance`, what happens if you don't include either option? Is there a default? If so, what is it? Or is one of these options required?
- Are there any other options that aren't listed?

Reply:
Every options I sow is here
https://github.com/Islandora/islandora_xacml_editor/blob/7.x/islandora_xacml_editor.drush.inc#L19

I made an info from that lines....

Reply:
@bondjimbond I corrected with your suggestions see it 
https://github.com/Islandora/islandora_xacml_editor/pull/142/files

Reply:
Good stuff. Just two small notes:
- For consistency, remove the "For example" on line 40
- Line 54: PIDs don't have datastreams, objects do.. Maybe reword for clarity - "The target object must have a POLICY datastream."

Reply:
@bondjimbond Done! Tnx for clearings and suggestion.

Reply:
Looking good! One more picky note: can you remove the period in the header on line 40? After that I can approve. :)

Reply:
Removed! Check it...